models/intel/device_support.md

```diff
@@ -60,6 +60,8 @@
 | machine-translation-nar-en-de-0002 | YES | YES |    |
 | machine-translation-nar-en-ru-0001 | YES | YES |    |
 | machine-translation-nar-ru-en-0001 | YES | YES |    |
+| machine-translation-nar-en-ru-0002 | YES | YES |    |
```

Comment:
probably if model gives on gpu 0.0 bleu, it can not be called supported for device?

Reply:
definetely, @bes-dev please leave only CPU as supported device

Reply:
Agree with you. Removed GPU support in device_support.md for all machine-translation-nar-* models.
Done